### ISSUE TITLE
Compiler error after latest platform.io update

### ISSUE THREAD

**Issue by hirschbrat** [alanswx/ESPAsyncWiFiManager#25]
Hello,

I used the library on an ESP32 (Lolin32) board. Since the latest platform.io update (espressif32 platform is now @ 1.1.2) I get the following errors:

```
In file included from .piolibdeps/ESPAsyncWifiManager_ID1438/ESPAsyncWiFiManager.cpp:14:0:
.piolibdeps/ESPAsyncWifiManager_ID1438/ESPAsyncWiFiManager.h: In constructor 'AsyncWiFiManager::AsyncWiFiManager(AsyncWebServer*
, DNSServer*)':
.piolibdeps/ESPAsyncWifiManager_ID1438/ESPAsyncWiFiManager.h:166:19: warning: 'AsyncWiFiManager::server' will be initialized aft
er [-Wreorder]
AsyncWebServer *server;

^
.piolibdeps/ESPAsyncWifiManager_ID1438/ESPAsyncWiFiManager.h:164:19: warning:   'DNSServer* AsyncWiFiManager::dnsServer' [-Wreor
der]
DNSServer      *dnsServer;
^
.piolibdeps/ESPAsyncWifiManager_ID1438/ESPAsyncWiFiManager.cpp:67:1: warning:   when initialized here [-Wreorder]
AsyncWiFiManager::AsyncWiFiManager(AsyncWebServer *server, DNSServer *dns) :server(server), dnsServer(dns) {
^
.piolibdeps/ESPAsyncWifiManager_ID1438/ESPAsyncWiFiManager.cpp: In member function 'void AsyncWiFiManager::scan()':
.piolibdeps/ESPAsyncWifiManager_ID1438/ESPAsyncWiFiManager.cpp:283:16: warning: unused variable 'res' [-Wunused-variable]
bool res=WiFi.getNetworkInfo(i, wifiSSIDs[i].SSID, wifiSSIDs[i].encryptionType, wifiSSIDs[i].RSSI, wifiSSIDs[i].BSSID, wifiSSIDs
[i].channel);
^
In file included from .piolibdeps/ESPAsyncWifiManager_ID1438/ESPAsyncWiFiManager.h:21:0,
from .piolibdeps/ESPAsyncWifiManager_ID1438/ESPAsyncWiFiManager.cpp:14:
.piolibdeps/ESPAsyncWifiManager_ID1438/ESPAsyncWiFiManager.cpp: In member function 'void AsyncWiFiManager::startWPS()':
/Users/to/.platformio/packages/framework-arduinoespressif32/tools/sdk/include/esp32/esp_wps.h:86:1: error: C99 designator 'manuf
acturer' outside aggregate initializer
}
^
.piolibdeps/ESPAsyncWifiManager_ID1438/ESPAsyncWiFiManager.cpp:590:29: note: in expansion of macro 'WPS_CONFIG_INIT_DEFAULT'
esp_wps_config_t config = WPS_CONFIG_INIT_DEFAULT(ESP_WPS_MODE);
^
/Users/to/.platformio/packages/framework-arduinoespressif32/tools/sdk/include/esp32/esp_wps.h:86:1: error: C99 designator 'model
_number' outside aggregate initializer
}
^
.piolibdeps/ESPAsyncWifiManager_ID1438/ESPAsyncWiFiManager.cpp:590:29: note: in expansion of macro 'WPS_CONFIG_INIT_DEFAULT'
esp_wps_config_t config = WPS_CONFIG_INIT_DEFAULT(ESP_WPS_MODE);
^
/Users/to/.platformio/packages/framework-arduinoespressif32/tools/sdk/include/esp32/esp_wps.h:86:1: error: C99 designator 'model
_name' outside aggregate initializer
}
^
.piolibdeps/ESPAsyncWifiManager_ID1438/ESPAsyncWiFiManager.cpp:590:29: note: in expansion of macro 'WPS_CONFIG_INIT_DEFAULT'
esp_wps_config_t config = WPS_CONFIG_INIT_DEFAULT(ESP_WPS_MODE);
^
/Users/to/.platformio/packages/framework-arduinoespressif32/tools/sdk/include/esp32/esp_wps.h:86:1: error: C99 designator 'devic
e_name' outside aggregate initializer
}
^
.piolibdeps/ESPAsyncWifiManager_ID1438/ESPAsyncWiFiManager.cpp:590:29: note: in expansion of macro 'WPS_CONFIG_INIT_DEFAULT'
esp_wps_config_t config = WPS_CONFIG_INIT_DEFAULT(ESP_WPS_MODE);
^
*** [.pioenvs/lolin32/lib2d4/ESPAsyncWifiManager_ID1438/ESPAsyncWiFiManager.cpp.o] Error 1
```

**Comment by JorgeChagui** [comment]
Hi, same error here.

**Comment by alanswx** [comment]
See if it works now. I don't love my fix..

**Comment by icandura** [comment]
I have same error, but detail is diffrent.

```
Compiling .pioenvs\nodemcuv2\src\main.cpp.o
Linking .pioenvs\nodemcuv2\firmware.elf
.pioenvs\nodemcuv2\libbd1\libESPAsyncWifiManager_ID1438.a(ESPAsyncWiFiManager.cpp.o):(.text._ZN16AsyncWiFiManager8startWPSEv+0x8): undefined reference to `WPS_is_unavailable_in_this_configuration__Please_check_FAQ_or_board_generator_tool'
.pioenvs\nodemcuv2\libbd1\libESPAsyncWifiManager_ID1438.a(ESPAsyncWiFiManager.cpp.o): In function `AsyncWiFiManager::startWPS()':
ESPAsyncWiFiManager.cpp:(.text._ZN16AsyncWiFiManager8startWPSEv+0x20): undefined reference to `WPS_is_unavailable_in_this_configuration__Please_check_FAQ_or_board_generator_tool'
collect2.exe: error: ld returned 1 exit status
*** [.pioenvs\nodemcuv2\firmware.elf] Error 1
```

**Comment by nadahmed** [comment]
downgrading the ESP core to 2.4.1 fixed this issue for me. Here's my platformio.ini code in atom.

`[env:nodemcuv2]
platform = espressif8266@1.7.3  ;change this line
board = nodemcuv2
framework = arduino
build_flags = -Wl,-Teagle.flash.1m64.ld
`

**Comment by icandura** [comment]
> downgrading the ESP core to 2.4.1 fixed this issue for me. Here's my platformio.ini code in atom.
> 
> > [env:nodemcuv2]
> > platform = espressif8266@1.7.3  ; <--------- change this line
> > board = nodemcuv2
> > framework = arduino
> > build_flags = -Wl,-Teagle.flash.1m64.ld

It works! Thank you!

> Release Notes
> 1.8.0 released about a month ago
> Add new boards: Amperka WiFi Slot, WiFiduino, SeeedStudio Wio Link, XinaBox CW01
> **Update Arduino core to 2.4.2**
> Switch to isolated build flags per framework
> Rename monitor_baud to monitor_speed
>    
> 1.7.3 released 3 months ago
> Fix SPIFFS uploading
>   
> 1.7.2 released 3 months ago
> Automatically calculate maximum available sketch size based on LD script (issue #86)
>   
> 1.7.1 released 3 months ago
> Declare command for checking memory usage
> Fix ROM size for WROOM-02
>   
> 1.7.0 released 5 months ago
> Allow switching between lwIP variants (issue #65)
> Update Arduino Core to 2.4.1 (issue #76)
> Update "esptool" to the latest version
> Update default upload speeds
> Fix to build a SPIFFS for 16MB

**Comment by alanswx** [comment]
I think this is fixed now.
